### DISCUSSION TITLE
Incorrect Number of Rows in Swiper Grid

### DISCUSSION THREAD

**Discussion by gajrajgchouhan** [nolimits4web/swiper#5552] [Q&A] [answered]
I had opened this issue #5542 , as the number of slides showing up was not correct, what is the correct way to show the slides? The example was taken from the [swiperjs.com/demos](https://swiperjs.com/demos) (Swiper Grid section)

**Comment by nolimits4web** [comment, marked as answer]
apparently you need to change slide's height to fit 3 slides

**Reply by gajrajgchouhan** [reply]
I checked and `grid: {rows: n}, spaceBetween: 0` + ` height: calc((100% - 0px) / n) !important;` works perfectly.

**Comment by jayna456** [comment]
@gajrajgchouhan, Not working in my case.

In my css:
.```
swiper-slide {
    border-right: 2px solid #ffffff;
    border-bottom: none;
    height: calc(100% - 108px) / 6 !important;
  }
```

and in JS:
```
new Swiper('.show-slider',{
grid: { rows: 6, fill: 'column' },
})
```

in html (angular):
```
<div class="show-slider" style="width: 538px;" id="indian-tour-slider" *ngIf="indian_tours.length">
<div class="show-slider-inner-wrap swiper-wrapper">
<div *ngFor="let indian_tour of indian_tours ; let i = index  " class="swiper-slide">
// slides in divs
</div>
</div>
</div>
```
            
Let me know where I am mistaken!
